app/src/main/java/com/besga/jonander/avisaaane/StarterApplication.java

```diff
@@ -17,18 +20,19 @@ public void onCreate() {
         Properties prop = new Properties();
         try {
             prop.load(getAssets().open("app.properties"));
-
-
         } catch (IOException ex) {
-            ex.printStackTrace();
+            Log.e(TAG, "File app.properties with ApplicationID and ClientKey not found", ex);
+            Intent homeIntent = new Intent(Intent.ACTION_MAIN);
```

Comment:
Calls the home screen and clears the activities stack, therefore, closing the whole app?